src/main/java/core/basesyntax/HelloWorld.java

```
@@ -1,9 +1,64 @@
 package core.basesyntax;
 
+import java.util.HashMap;
+import java.io.*;
+import java.util.*;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
-}
+    public static class Main {
+        public static void main(String[] arg) {
+            // 1. Read the data from the input CSV file
+            String inputFile = "input.csv";
+            String outputFile = "output.csv";
+
+            Map<String, Integer> fruitShop = new HashMap<>();
+
+            try (BufferedReader read = new BufferedReader(new FileReader(inputFile))) {
+                String line;
+                while ((line = read.readLine()) != null) {
+                    String[] parts = line.split(",");
+                    String operation = parts[0].trim();
+                    int quantity = Integer.parseInt(parts[2].trim());
+
+                    int current = fruitShop.getOrDefault(operation, 0);
+
+                    switch (operation) {
+                        case "b":
+                            fruitShop.put(operation, quantity);
+                            break;
+                        case "s":
+                            fruitShop.put(operation, current + quantity);
+                            break;
+                        case "p":
+                            fruitShop.put(operation, current - quantity);
+                            break;
+                        case "r":
+                            fruitShop.put(operation, current + quantity);
+                            break;
+                    }
+                }
+            } catch (IOException e) {
+                e.printStackTrace();
```

According to the checklist, you should not ignore exceptions in `try-catch` blocks. Instead of `e.printStackTrace()`, rethrow a `RuntimeException` with an informative message and the exception object. For example:
```java
throw new RuntimeException("Error reading from file", e);
```